ckan-client-guide/README.md

```diff
@@ -4,113 +4,523 @@ Guide to interacting with [CKAN](/ckan) for power users such as data scientists,
-[c2f]: https://github.com/datopian/frictionless-ckan-mapper-js
+::::tip
+
+As CKAN moves to Frictionless to default this will gradually become unnecessary.
+
+::::
 
 ## Quick start
 
-This guide currently uses a Python idiom by default (e.g. lower case, underscore). You can adapt to R and Javascript as needed and we provide alternative examples where possible.
+Most of this guide has Python programming language in mind, including its [convention regading using _snake case_ for instances and methods names](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles).
+
+If needed, you can adapt the instructions to JavaScript and R (coming soon) by using _camel case_ instead — for example, if in the Python code we have  `client.push_blob(…)`, in JavaScript it would be `client.pushBlob(…)`.
 
 ### Prerequisites
 
-Install the SDK for your language and configure it.
+Install the client for your language of choice:
 
-* Python: https://github.com/datopian/ckan3-py-sdk#install) 
-* Javascript: ...
-* R: coming soon
+* For Python, [use `pip install ckan-client`](https://github.com/datopian/ckan-client-py#install)
+* For JavaScript, [use `npm install https://github.com/datopian/ckan-client-js.git`](https://github.com/datopian/ckan-client-js#install)
+* R (_coming soon_)
 
-### Create a client
+### Create a client
+
+#### Python
 
 ```python
 from ckanclient import Client
 
-endpoint = 'https://my-ckan.com/'
-token = 'xxxx'              # your CKAN API key
-org_name = 'my-org'         # default organization on CKAN to add datasets to
-client = Client(endpoint, token, org_name)
+
+api_key = '771a05ad-af90-4a70-beea-cbb050059e14'
+api_url = 'http://localhost:5000'
+organization = 'datopian'
+dataset = 'dailyprices'
+lfs_url = 'http://localhost:9419'
+
+client = Client(api_url, organization, dataset, lfs_url)
+```
+
+#### JavaScript
+
+```javascript
+const { Client } = require('ckanClient')
+
+apiKey = '771a05ad-af90-4a70-beea-cbb050059e14'
+apiUrl = 'http://localhost:5000'
+organization = 'datopian'
+dataset = 'dailyprices'
+
+const client = Client(apiKey, organization, dataset, apiUrl)
 ```
 
-### Upload a resource (file) (and implicitly create a new Dataset)
+### Upload a resource
+
+That is to say, upload a file, implicitly creating a new dataset.
+
+#### Python
 
 ```python
-# loads a resource from a path
-resource = f11s.load(resource_file_path)
-dataset_name = 'sample-dataset'
-dataset = f11s.Dataset({'name': dataset_name})
-dataset.add_resource(resource)
-client.push(resource)
+from frictionless import describe
+
+
+resource = describe('my-data.csv')
+client.push_blob(resource)
 ```
 
 ### Create a new empty Dataset with metadata
 
+#### Python
+
 ```python
-dataset = f11s.Dataset({
-    'name': dataset_name,
-    'metadata': {'maintainer_email': 'sample@datopian.com'}
-})
-client.push(dataset)
+client.create('my-data')
+client.push(resource)
 ```
 
 ### Adding a resource to an existing Dataset
 
+::::tip NOTE
+Not implemented yet.
```

Comment:
Why is this not implemented yet? I think we could write this out so it was implemented.